bluemira/builders/EUDEMO/reactor.py

```diff
@@ -77,14 +92,58 @@ def build_plasma(self, **kwargs):
         default_eqdsk_name = f"{self._params.Name.value}_eqref.json"
         default_eqdsk_path = os.path.join(default_eqdsk_dir, default_eqdsk_name)
 
-        plasma_config = {
-            "name": name,
-            "plot_flag": self._build_config.get("plot_flag", False),
-            "run_mode": self._build_config.get("plasma_mode", "run"),
-            "eqdsk_path": self._build_config.get("eqdsk_path", default_eqdsk_path),
-        }
+        default_config = {"runmode": "run", "eqdsk_path": default_eqdsk_path}
 
-        builder = PlasmaBuilder(self._params.to_dict(), plasma_config)
+        config = self._process_design_stage_config(name, default_config)
+
+        builder = PlasmaBuilder(self._params.to_dict(), config)
         self.register_builder(builder, name)
 
         return super()._build_stage(name)
+
+    def build_TF_coils(self, component_tree: Component, **kwargs):
```

Comment:
kwargs are unused here. Is it needed for future dev?

Same with the above two functions. I think they should only be added if they are used right now (we can always add them if needed)